code/obj/item/storage/weapon_security.dm

```diff
@@ -326,6 +326,12 @@
 	/obj/item/boomerang,\
 	/obj/item/ammo/bullets/tranq_darts/syndicate = 4)
 
+/obj/item/storage/box/poisonbundle
+	name = "\improper Poison bottle box"
+	desc = "A box filled with random poison bottles."
```

Comment:
make the name/description a little ambiguous please (in the same manner as poison bottles themselves), so that a spaceperson (on RP) could reasonably see it as something other than 'oh yes this is/was a box full of poison' at a glance

Reply:
alright, I've come up with something